src/modules/navigator/mission.cpp

```diff
@@ -1048,12 +1048,14 @@ Mission::set_mission_items()
 		pos_sp_triplet->next.valid = false;
 	}
 
-	/* Save the distance between the current sp and the previous one */
-	if (pos_sp_triplet->current.valid && pos_sp_triplet->previous.valid) {
+	/* Save the distance between the current sp and our current position */
+	if (pos_sp_triplet->current.valid) {
 
 		_distance_current_previous = get_distance_to_next_waypoint(
```

Comment:
Drop "previous" from `_distance_current_previous` to reflect the change.

Reply:
Do you think _altitude_foh_distance would be good? I think it would describe its use better.